modules/ROOT/pages/release-notes.adoc

```diff
@@ -5,7 +5,7 @@ include::attributes.adoc[]
 :gs-link: https://integr8ly.github.io/user-documentation/index.html?e=modules/ROOT/pages/getting-started.adoc
 
 
-= Release Notes
+= Release Notes for Managed Integration {MINOR_VERSION_NO}
```

Comment:
not sure if the version no should be there, isn't that pulled into title automagically, check in Pantheon